client/src/Movies/MovieList.js

```diff
@@ -34,21 +36,8 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
-
-      {stars.map(star => (
-        <div key={star} className="movie-star">
-          {star}
-        </div>
-      ))}
-    </div>
+    <Link to={`/moveis/${movie.id}`}>
```

Comment:
In ` class MovieList`. In line 27, First delete everything under the `return` and paste this snippet

```
 <div className="movie-list">
        {this.state.movies.map(movie => (
          <div>
            <Link to={`/movies/${movie.id}`}>
            <MovieDetails key={movie.id} movie={movie} />
            </Link>
          </div>
        ))}
      </div>
```

this is where I want you to use Link because it's the only place you could get `movies.id` from.